about/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="ru">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Об авторе</title>
+  </head>
+  <body>
+    <h1>О себе</h1>
+    <p>
+      Меня зовут Даша. Мне <sub>22</sub> года. Закончила в 2021 году БГУИР по
+      специальности <strong>"Экономика электронного бизнеса"</strong>.<br />На
+      данный момент работаю в сфере <del>IT</del>
+      <ins>гемблинга.</ins> &#128049;
+    </p>
+    <h2>Мой путь в IT</h2>
+    <p>
+      Когда-то давным-давно я начала заниматься изучением Frontend разработки
+      самостоятельно, потому что как мне кажется это достаточно интересное и
+      востребованное направление в IT.<br />
+      Вдогонку к принятию этого решения стало то, что
+      <em>JavaScript</em> нелогичен, я не очень логичная, а следовательно у нас
+      все должно быть хорошо.
+    </p>
+
+    <blockquote>
+      <mark>Два сапога — пара. Если они одного размера и фасона.</mark>
+    </blockquote>
+    <p>
+      На данный момент мне очень интересно продолжить развиваться в данном
+      направлении, но уже при помощи курсов от Intellect Fox, так как
+      квалифицированные специалисты, прошедшие весь путь от <sup>"А"</sup> до
+      <sup>"Я"</sup>, смогут корректно донести информацию и помочь быстрее
+      влиться в профессию. &#128151;
+    </p>
+
+    <h3>Контактная информация</h3>
+    Gmail:<a href="mailto:korzenok.darya@gmail.com"
```

Comment:
Please add text into any tag or in `<a>`, it's a bad practice to live text just in `<body>`